### PR TITLE
HTML Changes to make it appealing

### PR DESCRIPTION
**Pull request by DEBSUBHRO**
Made some changes to make it look better.

## Changes 
- Centering the heading
- Centering the badge
- Linking the logo image
- Added NuGet badge and "Other" article heading.
- Added longtext-field to docs 
- Added number-field to docs 

## Issues
#329 
#330

### CODE REVIEW

File: _docs/longtext-field.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Integer Field"
```
Comment (herflis) [inline _docs/longtext-field.md]:
```suggestion
title: "LongText Field"
```

File: _docs/longtext-field.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+source_url: 'https://github.com/SenseNet/sensenet/blob/master/docs/integer-field.md'
```
Comment (herflis) [inline _docs/longtext-field.md]:
```suggestion
source_url: 'https://github.com/SenseNet/sensenet/blob/master/docs/longtext-field.md'
```

File: _docs/longtext-field.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- Field control is rendered as simple textarea
+
+## Related links
+
+- [Field](/field)
+- [longText Field Control](/longtext-field-control)
+
+# References
+
+[Category](/special:categories): [Field](/category:fields)
```
Comment (herflis) [inline _docs/longtext-field.md]: you can remove this part

File: _docs/number-field.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+---
+title: "Integer Field"
```
Comment (herflis) [inline _docs/number-field.md]:
```suggestion
title: "Number Field"
```

File: _docs/number-field.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+source_url: 'https://github.com/SenseNet/sensenet/blob/master/docs/integer-field.md'
```
Comment (herflis) [inline _docs/number-field.md]:
```suggestion
source_url: 'https://github.com/SenseNet/sensenet/blob/master/docs/number-field.md'
```

File: _docs/number-field.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+- the data is displayed with 4 decimal numbers
+
+## Related links
+
+- [Field](/field)
+- [Number Field Control](/number-field-control)
+
+# References
+
+[Category](/special:categories): [Field](/category:fields)
```
Comment (herflis) [inline _docs/number-field.md]: you can remove this part

**Comment by herflis** [comment]
Hey @DEBSUBHRO ,
thank you for your contribution! 🙏 I've added some minor suggestions to change, but otherwise it is ok. Don't forget to register 👉 [here](https://sensenet.com/Hacktoberfest2019)

**Comment by DEBSUBHRO** [comment]
> Hey @DEBSUBHRO ,
> thank you for your contribution! 🙏 I've added some minor suggestions to change, but otherwise it is ok. Don't forget to register 👉 [here](https://sensenet.com/Hacktoberfest2019)

Ma'am, the changes have been made. Please check them.

**Comment by herflis** [comment]
Thank you @DEBSUBHRO

**Pull request merged**